### PR TITLE
tf-doc add logging env variables for the terraform runner

### LINKED ISSUE #3236: tf doc - documenting logging-related environment variables of TF-controller
As a user,
I'd like to know details about logging-related environment variables used by TF-controller and its runner.

### PR DESCRIPTION
**Pull request by chanwit**
Fixes #3236 

Signed-off-by: Chanwit Kaewkasi <chanwit@weave.works>

### CODE REVIEW

File: website/docs/terraform/environment-variables.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Logging Env Variables
```
Comment (makkes) [inline website/docs/terraform/environment-variables.mdx:2]: We should be linking to this page from the appropriate context in the overview page.

Comment (chanwit) [reply website/docs/terraform/environment-variables.mdx:2]: It is listed automatically on the side bar. don't worry.

Comment (makkes) [reply website/docs/terraform/environment-variables.mdx:2]: Yeah, I mean like linking from it with a description along the lines of "for further configuration options refer to this page". This gives it a little more visibility.

Comment (chanwit) [reply website/docs/terraform/environment-variables.mdx:2]: Unfortunately, what you're saying is not fit nicely with the content of the overview page. Let me think about it next time as I'm also planning to re-org this doc somehow in the near future.

Comment (makkes) [reply website/docs/terraform/environment-variables.mdx:2]: Sure thing. We can definitely iterate. This PR is a great addition!! ✨

**Review by JamWils — APPROVED** [review]
LGTM.  Thanks @chanwit

**Pull request merged**